org.eclipse.jdt.ls.debug/pom.xml

```diff
@@ -36,6 +36,11 @@
                             <artifactId>reactive-streams</artifactId>
                             <version>1.0.0</version>
                         </artifactItem>
+                        <artifactItem>
```

Comment:
Don't add the commons-lang3 dependency from the pom.xml. It's already available in the target platform (.core uses it). You need to define the dependency in META-INF/MANIFEST.MF instead